webapp/api_handlers.go

```diff
@@ -48,6 +49,9 @@ func apiTestRunsHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	labels := ParseLabelsParam(r)
```

Comment:
I think we should have comments somewhere discussing the binding between `unstable` in one context and `-experimental` in another.

Reply:
@jugglinmike - can you weigh in on your preference or `stable` or `experimental` as the label? You chose `-experimental` as the browser suffix, so perhaps you prefer that? I'm not fussed - unstable does, as @foolip points out, kinda sound test-related.

Reply:
Yup, I agree with @foolip's advice, and since he made the improvement to WPT, using the same convention here will promote consistency in terminology.

Reply:
OK - `unstable` => `experimental`